pulsar/consumer.go

```diff
@@ -278,6 +278,11 @@ type Consumer interface {
 	// where more than one consumer are currently connected.
 	UnsubscribeForce() error
 
+	// GetLastMessageIDs get all the last message id of the topics the consumer subscribed.
+	//
+	// The list of MessageID instances of all the topics that the consumer subscribed
+	GetLastMessageIDs() ([]MessageID, error)
```

Comment:
We need to implement a TopicMesasgeID and return TopicMessageIDs here. Otherwise, the users can't know which  messages ids belong to which topic.

Reply:
I was careless, done.